object-group-game/Inventory/Item.cs

```diff
@@ -3,17 +3,24 @@
 using System.Linq;
 using System.Text;
 
-namespace object_group_game 
+namespace object_group_game
 {
-	class Item
+	public class Item
 	{
-		public string Name { get; private set; }
-		public List<Effect> Effects { get; private set; }
+		public int ID { get; set; }
+		public string DisplayName { get; set; }
+		public List<Effect> Effects { get; set; }
 
-		Item(string name, ICollection<Effect> effects)
+		public Item (Database.Item item, List<Effect> effects)
```

Comment:
This class shouldn't depend on the Database, you should be able to create Items without connection to the Database or anything connected to Database.

Reply:
The issue is that I should be able to create the item from the code using new Item(parameters), and later add it to the Database. But with this approach, it seems that I have to have some sort of Database.Item beforehand.